### PR TITLE
Translated traffic-mirroring-zh.

### PR DESCRIPTION
**Pull request by Patrick-LuoYu**
Signed-off-by: Patrick-LuoYu <patrickluo@yunify.com>
/assign @Sherlock113

### CODE REVIEW

File: content/zh/docs/project-user-guide/grayscale-release/traffic-mirroring.md [hunk 1]
```diff
@@ -1,74 +1,68 @@
...
 
-   - When you execute the command above, replace `demo-project` with your own project (i.e. namespace) name.
-   - If you want to execute the command from the web kubectl on the KubeSphere console, you need to use the account `admin`.
+   - 请将上述命令中的 `demo-project` 修改成实际的项目（即 Namespace）名称。
+   - 您需要以 `admin` 用户重新登录才能在 KubeSphere 控制台的 Web kubectl 页面执行上述命令。
 
    {{</ notice >}} 
 
-10. Expected output:
+10. 命令执行结果：
```
Comment (Sherlock113) [inline content/zh/docs/project-user-guide/grayscale-release/traffic-mirroring.md:65]: 命令执行结果：
->
预期输出结果：

File: content/zh/docs/project-user-guide/grayscale-release/traffic-mirroring.md [hunk 2]
```diff
@@ -1,74 +1,68 @@
...
 
    ```bash
    kubectl -n demo-project get virtualservice -o yaml
    ```
 
    {{< notice note >}} 
 
-   - When you execute the command above, replace `demo-project` with your own project (i.e. namespace) name.
-   - If you want to execute the command from the web kubectl on the KubeSphere console, you need to use the account `admin`.
+   - 请将上述命令中的 `demo-project` 修改成实际的项目（即 Namespace）名称。
```
Comment (Sherlock113) [inline content/zh/docs/project-user-guide/grayscale-release/traffic-mirroring.md:60]: 即 Namespace
->
即命名空间

**Review by Sherlock113 — COMMENTED** [review]
Have a look at my comments and make changes. Thanks.

**Comment by Patrick-LuoYu** [comment]
Fixed issues based on comments.
/assign @Sherlock113

**Pull request merged**